app/app.go

```diff
@@ -324,6 +342,14 @@ type App struct {
 
 	// sm is the simulation manager
 	sm *module.SimulationManager
+
+	// Custom checkTx handler -> this check-tx is used to simulate txs that are
+	// wrapped in a bid-tx
+	checkTxHandler checktx.CheckTx
+
+	// MEVLane is used as a dependency for the x/auction module's antehandler. This references th lane in the app's
+	// LaneMempool.
+	MEVLane auctionante.MEVLane
```

Comment:
Not sure we need to define a field for `auctionante.MEVLane`, the only place you use it is L1000, passing it to antehandler, but you can do it without an app's field